addons/web_editor/static/lib/odoo-editor/src/powerbox/patienceDiff.js

```diff
@@ -0,0 +1,263 @@
+/** @odoo-module **/
```

Comment:
C'est ok de foutre les libs dans `src` comme ça et pas dans un sous-dossier `lib` ?

Reply:
Yes. That comes from the organisation discussed in private.

Reply:
Also, that's just a function, it cannot really be considered as a lib even tho the code originally comes was created externally.